app/index.html

```diff
@@ -21,7 +80,16 @@
 		}
 	</script>
 
-	<main>Loading...</main>
+	<main></main>
+
+	<section class='splash-screen'>
+		<h1>Lunaria</h1>
+		<svg xmlns='http://www.w3.org/2000/svg' width='24' height='24' viewBox='0 0 24 24' fill='currentColor'>
+			<circle cx='4' cy='12' r='3' />
+			<circle cx='12' cy='12' r='3' />
+			<circle cx='20' cy='12' r='3' />
+		</svg>
+	</section>
```

Comment:
If you put this in `main`, and have the fragment returned by the `App` component have a `main` for its root element, then preact will replace this with the app once React is ready to take over rendering.  This way you don't have to do the `useSplashEffect` thing to hide it.

Reply:
this was to add a subtle transition especially when refreshing the page which makes the page looks like it "flickers" and can be confusing for the user (they saw something happened but was too quick to visually read/analyze it). the effect should hide/reduce it by the fading out the splash screen positioned over the `main`.


![splash-screen-2](https://user-images.githubusercontent.com/1169838/213334484-cd4b6b57-63e4-4ea1-b540-b2398381b301.gif)

Reply:
Ah, OK.  My ability to read CSS is very poor, and I thought the transition was instant.